misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/ResultFeatureAssertionsSpec.kt

```diff
@@ -81,8 +89,9 @@ abstract class ResultFeatureAssertionsSpec(
         }
     }
 
-    describeFun(isSuccess.name /* TODO #204: , "${isFailureFeature.name}" */) {
+    describeFun(isSuccess.name, isFailureFeature.name) {
```

Comment:
my fault, should have been `isFailure` and not `isFailureFeature` please re-add `isFailure as in your first commit